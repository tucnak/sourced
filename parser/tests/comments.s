// This is a single-line comment, which must be absolutely ignored
r_drawviewmodel 1

// Yet another comment here!
fov_desired 90
