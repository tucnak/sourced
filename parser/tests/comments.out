r_drawviewmodel 1
fov_desired 90
